probe-rs-target/src/flash_algorithm.rs

```diff
@@ -97,6 +97,14 @@ pub struct RawFlashAlgorithm {
     /// The encoding format accepted by the flash algorithm.
     #[serde(default)]
     pub transfer_encoding: Option<TransferEncoding>,
+
+    /// `true` if the instructions are saved in Big Endian format
+    #[serde(default = "no")]
```

Comment:
This is just `serde(default)`.

Reply:
Removed

Reply:
(Originally it was called `little_endian` to match the `object` crate's opinion, but I inverted it so that all existing flash algorithms would be correct.)